app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_comments_counter
+    post.comments_counter += 1
+  end
+end
```

Comment:
- [ ] [OPTIONAL] As I have recommended above, I would like to again suggest that you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private.

```ruby
after_save :update_comments_counter

  private

  def increment_comment_counter
   Code here ...
  end
```

Reply:
Thank you very much for your beneficial guide @Whoistolu I will add the recommended codes.